lib/annotate/annotate_models.rb

```diff
@@ -904,6 +904,17 @@ def classified_sort(cols)
       ([id] << rest_cols << timestamps << associations).flatten.compact
     end
 
+    def map_col_type_to_ruby_classes(col_type)
```

Comment:
Just curious - can you elaborate why it should not be private and public instead?

Reply:
I don't know, my first idea was that it can be reused somewhere else but seems not yet so i'll add it in private